### PR TITLE
fix: docker container did not build issue-890

### PR DESCRIPTION
**Pull request by chrisjohnson00**
This should fix the problems I outlined in issue #890

### CODE REVIEW

**Comment by chrisjohnson00** [comment]
@gregorycoleman - for some reason GH won't allow me to use the re-request button... nothing happens, so please re-review at your convenience.

File: requirements.txt [hunk 1]
```diff
@@ -33,5 +33,5 @@ seaborn==0.10.1
 six==1.14.0
 toml==0.10.0
 typed-ast==1.4.1
-urllib3==1.26.5
```
Comment (0xdabbad00) [inline requirements.txt:36]: I want to lock CloudMapper to specific releases.

Comment (chrisjohnson00) [reply requirements.txt:36]: Agree with you on this!
I've rebased the conflicts and the docker build works now with the `requirements.txt` from the `main` branch.

**Comment by chrisjohnson00** [comment]
Ready for re-review, but the re-review button is disabled ;)

**Comment by 0xdabbad00** [comment]
Thank you @chrisjohnson00

**Pull request merged**